cognite/client/data_classes/datapoints.py

```diff
@@ -386,7 +511,9 @@ class Datapoint(CogniteResource):
         value (str | float | None): The raw data value. Can be string or numeric.
         average (float | None): The time-weighted average value in the aggregate interval.
         max (float | None): The maximum value in the aggregate interval.
+        max_datapoint (MaxDatapoint | MaxDatapointWithStatus | None): No description.
```

Comment:
Note to self: update
```suggestion
        max_datapoint (MaxDatapoint | MaxDatapointWithStatus | None): No description.
```